src/main/java/io/axoniq/axonserver/connector/event/PersistedStreamProperties.java

```diff
@@ -0,0 +1,46 @@
+package io.axoniq.axonserver.connector.event;
+
+import java.util.List;
+
+public class PersistedStreamProperties {
+    private final String streamName;
+    private final int segments;
+    private final String sequencingPolicyName;
+    private final List<String> sequencingPolicyParameters;
+    private final long initialPosition;
+    private final String filter;
+
+    public PersistedStreamProperties(String streamName, int segments, String sequencingPolicyName,
```

Comment:
Javadoc is missing on this class.

Is `0` a valid value for `segments`? If not, would be good to validate that.